tools/vmd/hbonds/hbonds.xml

```diff
@@ -0,0 +1,79 @@
+<tool id="vmd_hbonds" name="Hydrogen Bond Analysis using VMD" version="1.9.3">
+    <description>Analyze H-bonds between two segments</description>
+    <requirements>
+        <requirement type="package" version="1.9.3">vmd</requirement>
+    </requirements>
+    <command detect_errors="exit_code">
+<![CDATA[
+     ln -s '$__tool_directory__/hbonds.tcl' ./hbonds.tcl &&
+     vmd -pdb '$strin' -dcd '$trajin' -dispdev text -e '$hbonds_inp' &>> verbose.txt 
+]]></command>
+    <configfiles>
+         <configfile name="hbonds_inp">
+source hbonds.tcl
+
+set sele1 [atomselect top "$sele1"]
+set sele2 [atomselect top "$sele2"]
+
```

Comment:
I *think* these should be single-quotes?

Reply:
Tried single-quotes did not work. The atom selections must be in double quotes in VMD.

Reply:
As an explanation, the double quotes should be avoided in everything that is put on the commandline. This is because double quotes are evaluated by bash ... single-quotes are not evaluated. This means that an attacker could construct a filename with a env var included or something like that and execute code. To prevent this we want to use single-quotes on CMD calls. In this case it is only in a config file right? So double quotes should be ok.

Reply:
No, I get why we use single quotes. Just a reflex action here I guess. :)

Reply:
@bgruening thanks! The double quotes only used in the config file.